molecule/confluent-kafka-kerberos-customcerts-rhel/molecule.yml

```diff
@@ -11,8 +11,8 @@ platforms:
     hostname: kerberos1.confluent
     groups:
       - kerberos_server
-    image: geerlingguy/docker-centos7-ansible
-    dockerfile: ../Dockerfile.j2
+    image: centos:centos8
```

Comment:
why are we using centos:centos8 here instead of ubi8? does centos:centos8 have arm64 image?

Reply:
Both ubi8 and centos8 have amd and arm support. The difference is with the package `openldap-servers` In centos this package is available in the powertools repository vs in redhat/ubi8 it is paid.